examples/USGS/USGS_NLCD_RELEASES_2023_REL_RCMAP_V6_COVER.js

```diff
@@ -0,0 +1,36 @@
+// Import the NLCD RCMAP collection.
+var dataset = ee.ImageCollection('USGS/NLCD_RELEASES/2023_REL/RCMAP/V6/COVER');

```

Comment:
you can just say

var nlcd2021 = ee.Image('USGS/NLCD_RELEASES/2023_REL/RCMAP/V6/COVER/2021');

Reply:
This line doesn't make sense to me. a. the Cover is not an Image - it's an Image Collection, and b. why would the 2021 be at the end of that path? 
I ran the js file, as is, in the code editor and it ran fine.

Reply:
An ImageCollection has Image assets as its children, so while your code is correct, it's slightly shorter to refer to one specific image in the collection by its full id.

Reply:
Gotcha, that makes sense. I tried running it commenting out lines 2 and 5 and adding in that line you suggested for line 3. Should I update the js to just the 1 line like you suggested? Thanks!